app/Http/Controllers/Api/ProjectController.php

```diff
@@ -44,88 +59,152 @@ public function add_project(Request $request){
         try{
             $result = Project::create([
                 'user_id'=> $request->user()->id,
-                'project_address_id' => $request->project_address_id,
+                'builder_category_id' => null,
+                'builder_subcategory_id' => null,
                 'forename' => $request->forename,
                 'surname' => $request->surname,
                 'project_name' => $request->project_name,
                 'description' => $request->description,
                 'contact_mobile_no' => $request->contact_mobile_no,
+                'contact_home_phone' => $request->contact_home_phone,
                 'contact_email' => $request->contact_email,
+                'postcode' => $request->postcode,
+                'county' => $request->county,
+                'town' => $request->town,
+                'categories' => $request->categories,
+                'subcategories' => $request->subcategories,
+                'customer_note' => null,
+                'tradeperson_note' => null,
+                'internal_note' => null,
+                'status' => $request->status,
+                'reviewer_id' => null,
+                'reviewer_status' => null,
+                'reviewer_status_updated_at' => null,
                 'notes'=> $request->notes
             ]);
             if (!$result) {
                 throw ValidationException::withMessages(['message' => 'Something went wrong, please try again!'], 400);
             }
+            foreach ($request->images as $file) {
+                $fileName = $file->getClientOriginalName();
+                $extension = $file->getClientOriginalExtension();
+                $s3FileName = Str::uuid().'.'.$extension;
+                $file_type = explode('/', mime_content_type($file->getRealPath()))[0];
+                Storage::disk('s3')->put('Testfolder/'.$s3FileName, file_get_contents($file->getRealPath()));
```

Comment:
Asked You previously to get the folder name from const file

Reply:
Changes to be made for both line 93 and 94

Reply:
resolved